tensorflow/python/keras/losses.py

```diff
@@ -646,7 +646,7 @@ class CategoricalHinge(LossFunctionWrapper):
   """Computes the categorical hinge loss between `y_true` and `y_pred`.
 
   `loss = maximum(neg - pos + 1, 0)`
-  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
+  where `neg = maximum((1 - y_true)*y_pred)` and `pos = sum(y_true * y_pred)`
```

Comment:
This is unrelated change. Please don't include unrelated changes.

Reply:
Okay